cmd/ike/cmd/root.go

```diff
@@ -38,14 +38,18 @@ func NewRootCmd() *cobra.Command {
 		},
 		RunE: func(cmd *cobra.Command, args []string) error { //nolint[:unparam]
 			printVersion()
+			justPrintVersion, _ := cmd.Flags().GetBool("version")
```

Comment:
Always print version, but you can choose to only print the version?

Reply:
I preserved original behaviour, as I thought it's helpful on the server side when the service starts.

Reply:
Alternatively, we can refactor this out and provide `serve` (or `start` or any better name) subcommand which will do operator thing in the cluster.

Reply:
Maybe having `develop` vs `serve` subcommand will make it more clear on what this tool provides (vs root which is used right now).

Reply:
I think a 'serve' sub command would work

Reply:
I will create new issue then and provide separated PR. Now back to this change - are we ok with merging it or we stick to `ike version`?

Reply:
#131